assets/fs.go

```diff
@@ -6,3 +6,12 @@ import (
 
 //go:embed *
 var EmbedFS embed.FS
+
+func GetBytesOrPanic(path string) []byte {
+	bytes, err := EmbedFS.ReadFile(path)
+	if err != nil {
+		panic(err)
+	}
+
+	return bytes
+}
```

Comment:
This function isn't intended to be used outside of tests, and it's merely a helper to avoid writing code like `bytes, err := assetsFs.ReadFile(path)`. There's also a test for it to get its code coverage.